db/request.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Game = require('./init').Game
+const Request = require('./init').Request
+
+function getRequestByGame (gameName) {
+  return new Promise((resolve, reject) => {
+    Game.find({'name': gameName}, (err, game) => {
+      if (err !== null && typeof err !== 'undefined') {
+        return reject(new Error(err))
+      }
+      console.log(game)
+      if (game.length > 1) {
+        return reject(new Error('Too many Games found'))
```

Comment:
So if there is multiple of a game in the database then it is a critical error that needs to be fixed on the development end, instead of 'Too many Games found' maybe something like 'There is a problem with the integrity of the database'. Also maybe make a TODO comment indicating that this is something that needs to be logged